### PR TITLE
Check Again > [Abbr]

### PR DESCRIPTION
**Pull request by idpack**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -65,6 +65,34 @@
...
+				"<link[^>]* href=\"[^\"]*abbr.css",
+				"<link[^>]* href=\"[^\"]*abbr.min.css",
+				"<link[^>]* href=\"[^\"]*abbr.imp.css",
+				"<link[^>]* href=\"[^\"]*abbr.imp.min.css",
+				"<link[^>]* href=\"[^\"]*abbr.onc.css",
+				"<link[^>]* href=\"[^\"]*abbr.onc.min.css",
+				"<link[^>]* href=\"[^\"]*abbr.onc.imp.css",
+				"<link[^>]* href=\"[^\"]*abbr.onc.imp.min.css"
+				]
+			"icon": "Abbr.png",
```
Comment (AlexByte) [inline src/apps.json]: comma is missing.

Comment (AlexByte) [inline src/apps.json]: Comma is missing.

File: src/apps.json [hunk 2]
```diff
@@ -65,6 +65,34 @@
...
+			"implies": "CSS",
```
Comment (AlexByte) [inline src/apps.json]: Where is no such detectable technology.

**Comment by AlexByte** [comment]
Another technology used only by the author.

**Comment by AliasIO** [comment]
Closing as this software does not appear to be widely used.

**Pull request closed without merging**